src/installation/integrations.md

```diff
@@ -50,14 +56,12 @@ const compiledTemplate = require("./template.handlebars");
 
 ## Parcel: parcel-plugin-handlebars
 
-There is an old plugin for parcel:
-https://www.npmjs.com/package/parcel-plugin-handlebars
+There is an old plugin for parcel: https://www.npmjs.com/package/parcel-plugin-handlebars
```

Comment:
by `gc -m '....'` will be started the `pre-commit` hook and reformat MD file automatically :-/